src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/tests/test_organizational_policy_campaigns.py

```diff
@@ -0,0 +1,1159 @@
+                        "Type": "SERVICE_CONTROL_POLICY",
+                        "AwsManaged": False,
+                    }
+                ]
+            },
+        )
+
+        stubber.add_response(
+            "list_policies_for_target",
+            {"Policies": []},
+            {"TargetId": "09876543210", "Filter": "SERVICE_CONTROL_POLICY"},
+        )
+
+        # Creates the second policy and attaches the policy to the target
+        stubber.add_response(
+            "create_policy",
+            {
+                "Policy": {
+                    "PolicySummary": {
+                        "Id": "fake-policy-id-2",
+                        "Arn": "arn:aws:organizations:policy/fake-policy-id-2",
+                        "Name": "MySecondPolicy",
+                        "Description": "ADF Managed scp",
+                        "Type": "SERVICE_CONTROL_POLICY",
+                        "AwsManaged": False,
+                    },
+                    "Content": "fake-policy-content",
+                }
+            },
+            {
+                "Content": ANY,
+                "Description": "ADF Managed scp",
+                "Name": "MySecondPolicy",
+                "Type": "SERVICE_CONTROL_POLICY",
+            },
+        )
+        stubber.add_response(
+            "attach_policy",
+            {},
+            {"PolicyId": "fake-policy-id-2", "TargetId": "09876543210"},
+        )
+
+        stubber.add_response(
+            "list_targets_for_policy",
+            {
+                "Targets": [
+                    {
+                        "TargetId": "11223344556",
+                        "Arn": "arn:aws:organizations:account11223344556",
+                        "Name": "MyThirdOrg",
+                        "Type": "ORGANIZATIONAL_UNIT",
+                    }
+                ]
+            },
+            {"PolicyId": "fake-policy-1"},
+        )
+
+        stubber.add_response(
+            "detach_policy",
+            {},
+            {"PolicyId": "fake-policy-1", "TargetId": "11223344556"},
+        )
+        stubber.add_response(
+            "delete_policy",
+            {},
+            {"PolicyId": "fake-policy-1"},
+        )
+        stubber.activate()
+
+        policy_campaign = OrganizationPolicyApplicationCampaign(
+            "SERVICE_CONTROL_POLICY", org_mapping, {}, org_client
+        )
+
+        for _policy in POLICY_DEFINITIONS[1:]:
+            policy = policy_campaign.get_policy(
+                _policy.get("PolicyName"), _policy.get("Policy")
+            )
+            self.assertEqual(0, len(policy.targets_requiring_attachment))
+            policy.set_targets(
+                [policy_campaign.get_target(t) for t in _policy.get("Targets")]
+            )
+            self.assertEqual(1, len(policy.targets_requiring_attachment))
+
+        policy_campaign.apply()
+        stubber.assert_no_pending_responses()
+
+
+class SadTestCases(unittest.TestCase):
```

Comment:
Can you add a test case that will test that we throw an error when AWS Organizations is not configured with all features that we require for Tagging and Service Control Policies?